src/core/messages.ts

```diff
@@ -199,6 +215,25 @@ export interface Messages extends EmitsDiscontinuities {
    */
   send(params: SendMessageParams): Promise<Message>;
 
+  /**
+   * Delete a message in the chat room.
+   *
+   * This method uses the Ably Chat API REST endpoint for deleting messages.
+   * It performs a `soft` delete, meaning the message is marked as deleted.
+   *
+   * Note that the Promise may resolve before OR after the message is deleted
+   * from the realtime channel. This means you may see the message that was just
+   * deleted in a callback to `subscribe` before the returned promise resolves.
+   *
+   * Should you wish to restore a deleted message, and providing you have the appropriate permissions,
```

Comment:
Perhaps worth a comment that this is subject to change?

Reply:
How about..
```
Note: This is subject to change in future versions, whereby a new permissions model will be introduced
and a deleted message may not be restorable in this way.
```

Reply:
SGTM!